CONTRIBUTING.md

```diff
@@ -96,6 +96,24 @@ specific service, for example:
 AWS_REGION=[set region] node node_modules/cucumber/bin/cucumber.js --tag @s3
 ```
 
+### Testing the documentation changes:
+
+To test the documentation changes, please ensure you have the following requirements installed:
+
+* `gem install bundler`
+
+Once the bundler is installed, you need to execute the following commands, to install rake:
+
+```
+bundle install
+```
+
+Once done, please run this command to get the documentation in (doc/latest/_index.html)
+
+```
+bundle exec rake docs:api
```

Comment:
This addition is helpful.  It is also noted in the PR template: https://github.com/aws/aws-sdk-js/blob/master/.github/pull_request_template.md

Reply:
Agreed, that will only be seen while we go to raise a PR. So thought it would be good for contribution folks to see this in Contribution file :)